crates/re_build_web_viewer/src/lib.rs

```diff
@@ -165,6 +167,17 @@ pub fn build(profile: Profile, target: Target, build_dir: &Utf8Path) -> anyhow::
     if profile == Profile::Release {
         eprintln!("Optimizing wasm with wasm-opt…");
 
+        if !std::process::Command::new("wasm-opt")
```

Comment:
This command and the one below should really just be using [`Command::output`](https://doc.rust-lang.org/std/process/struct.Command.html#method.output) and dump the child's stderr if `!success()`.